services/http/api/lib/models/measurement.ts

```diff
@@ -0,0 +1,18 @@
+import { ModelRelatedNodesI, NeogmaInstance, NeogmaModel } from "neogma";
+import { UserModel, UserInstance } from "./user";
+import { DateInstance, DateModel } from "./date";
+
+export type MeasurementProperties = {
+    type: string,
+    value: number
+}
+
+export type MeasurementRelatedNode = {
+    User: ModelRelatedNodesI<UserModel, UserInstance>,
+    Date: ModelRelatedNodesI<DateModel, DateInstance>
```

Comment:
Oh I see. Are these types though or simply fields in the related nodes object? If it is the latter, I think they should be lower-case to avoid confusion.

Reply:
Sounds like a good idea, I'll implement them.

Reply:
@rlnsy Turns out, I was dead wrong. Apparently for neogma to recognize the models being related, the ModelRelatedNode types have to exactly match the model label. I reverted the changes.